resources/views/graph.blade.php

```diff
@@ -1,3 +1,10 @@
+<!--This is the result page for the website of the filled in workflows. The result page shows the result of the Evidencio API call.
```

Comment:
I'd suggest changing the comments to blade template's comments ( https://laravel.com/docs/5.6/blade#comments ) so they are not rendered in the final document. Now the comments show up above the whole HTML code.